### ISSUE TITLE
Backoffice Restrict Public Access link 404'ing

### ISSUE THREAD

**Issue by jattwood** [umbraco/Umbraco-CMS#12167]
### Which *exact* Umbraco version are you using? For example: 9.0.1 - don't just write v9

9.4 RC

### Bug summary

When trying to restrict access to a page in the backoffice on 9.4 RC, the blade 404's with:

Request error: The URL returned a 404 (not found):
/umbraco/backoffice/umbracoapi/content/GetPublicAccess

### Steps to reproduce

Right mouse click on a node to protect. Select Restrict Public Access.

### Expected result / actual result

Expect the public access blade to slide out but just getting:

![image](https://user-images.githubusercontent.com/6452616/159723388-ace277cf-88d0-4832-847f-19387b5aef86.png)

**Comment by nul800sebastiaan** [comment]
@jattwood Is this on Windows or Linux?

**Comment by jattwood** [comment]
Hey @nul800sebastiaan, good question. This is actually deployed in dev currently using IIS set up with a custom domain using the local hosts file with a self-signed (but trusted) cert, and VS Development Time Support for IIS. Everything else is working fine (I was just working through another not implemented exception found in Autolinking using Auth0 which was fixed in 9.4 RC when I stumbled on this now)

I do have another 9.3 site running currently under the exact same config as above with no issues with the Restrict Public Access

Thanks!
